lib/init.coffee

```diff
@@ -1,8 +1,56 @@
+{CompositeDisposable} = require 'atom'
+
 module.exports =
   config:
-    xmllintExecutablePath:
+    executablePath:
       type: 'string'
-      default: ''
-
+      title: 'xmllint Executable Path'
+      default: 'xmllint'
   activate: ->
     console.log 'activate linter-xmllint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-xmllint.xmllintExecutablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+    console.log 'activate: ' + @executablePath
```

Comment:
Although I'm not strictly against this, most providers have moved away from sending a message to the console on activate.

If you keep this use the linter name, not the executable path.